src/resources/index.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
-const colors = require('colors/safe');
+const { green, yellow } = require('colors/safe');
+
```

Comment:
I moved things around in this file to optimise readability, definitely doing another separate PR afterwards to reorganise this stuff